mlflow/entities/__init__.py

```diff
@@ -40,8 +40,8 @@
 from mlflow.entities.span_status import SpanStatus, SpanStatusCode
 from mlflow.entities.trace import Trace
 from mlflow.entities.trace_data import TraceData
-from mlflow.entities.trace_info import TraceInfo
-from mlflow.entities.trace_info_v3 import TraceInfoV3
+from mlflow.entities.trace_info import TraceInfo as TraceInfoV2
```

Comment:
do we want to change `mlflow.entities.trace_info` to `mlflow.entities.trace_info_v2`?